test_util/test_upgrade_vpc.py

```diff
@@ -195,7 +213,7 @@ def apply(self, dcos_url: str, masters: List[str], public_masters: List[str], sl
         pass
 
 
-class VpcClusterUpgradeTest:
+class VpcClusterUpgradeTestFactory:
```

Comment:
I dont think this needs to change to a factory so mesos_metrics_snapshot can be overriden. The downstream can just inherit this class and override. Am I missing something?